src/overlays/actors/ovl_En_Ma4/z_en_ma4.c

```diff
@@ -663,7 +663,7 @@ void EnMa4_DialogueHandler(EnMa4* this, GlobalContext* globalCtx) {
 
         case 6: // End conversation
             if (func_80147624(globalCtx) != 0) {
-                if ((globalCtx->msgCtx.unk_120B1 == 0) ||
+                if ((globalCtx->msgCtx.unk120B1 == 0) ||
```

Comment:
In OoT, it's been consistent to have underscores between the `unk` and the number. In MM, I haven't been too picky about it but I should bring it up that it's generally been inconsistent in MM atm

Reply:
MM is supposed to not have the underscore.